src/ice_shelf/MOM_ice_shelf.F90

```diff
@@ -1962,9 +1962,6 @@ subroutine initialize_ice_shelf(param_file, ocn_grid, Time, CS, diag, Time_init,
       'ice shelf thickness', 'm', conversion=US%Z_to_m)
   CS%id_dhdt_shelf = register_diag_field('ice_shelf_model', 'dhdt_shelf', CS%diag%axesT1, CS%Time, &
       'change in ice shelf thickness over time', 'm s-1', conversion=US%Z_to_m*US%s_to_T)
-  CS%id_mass_flux = register_diag_field('ice_shelf_model', 'mass_flux', CS%diag%axesT1,&
-      CS%Time, 'Total mass flux of freshwater across the ice-ocean interface.', &
-      'kg/s', conversion=US%RZ_T_to_kg_m2s*US%L_to_m**2)
 
```

Comment:
Was the removal of the `CS%id_mass_flux = register_diag_field()` call intentional?  If so, the corresponding `post_data(CS%id_mass_flux, ...)` call on line 849 should also be removed, and a comment explaining why should be added to the description of the commit.